### PR TITLE
1 year

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-22-one-year.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+author: Valentino Urbano
+---
+
+365 days have passed since I started writing one post every day.
+
+It is impressive to see how it all compounds. Most of the visits to my website nowadays are from posts written this past year, even though most of them are relatively short.
+
+I had to choose between consistency and writing long articles and I chose the former. Writing long form articles gives way better results with search engines and thanks to that you get more traffic, but that doesn't mean that short content is bad. It will just give you less views coming from Google.
+
+That said, I've always been a believer that if you're writing to get views or rank on Google you are better doing something else with your time. If you write content for your day job as a way to get leads it is another story, but if you're writing as an hobby, the main goal should be the craft itself and the focus should be on improving your skills. The rest should come naturally on its know, if it does, and it should not be the goal.
```
Comment (MsSlacker) [inline _posts/2019-11-22-one-year.md]: on its own

File: _posts/2019-11-22-one-year.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+## It Takes Time
+
+Change takes time.
+
+For a while you might see yourself not improving. It is not something you notice on a day to day basis. It is only when looking back after 6 months to a year that you will see a difference on how your approached things back then and how you approach them now. This is true not only for writing, but more in general. You should not mistake this for "it gets easier", it does not. Writing is still hard and will always be hard. The difference is in how you approach it. After a while you establish a process from the idea to the published article.
```
Comment (MsSlacker) [inline _posts/2019-11-22-one-year.md]: Instead of 6 month (...) - > after a longer period of time?

Comment (MsSlacker) [reply _posts/2019-11-22-one-year.md]: see - > notice?

Comment (MsSlacker) [reply _posts/2019-11-22-one-year.md]: your - > you

File: _posts/2019-11-22-one-year.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+## Consistency
+
+There has been days when I was sick, days when I was abroad, days when I was driving 12 hours straight (do not recommend), days with no internet connectivity[^1], days when I wanted to write, days when I didn't want to write, days when I was tired. There will always be an excuse to quit. You either quit or say not today and write. There is no secret for consistency it is just showing up every day to write. However says it is easy has either never done it or is lying. It is not easy, as with every commitment there are moments when you would rather do something else. But it has his highlights as well (such as today).
```
Comment (MsSlacker) [inline _posts/2019-11-22-one-year.md]: has - > have

Comment (MsSlacker) [reply _posts/2019-11-22-one-year.md]: excuseS

Comment (MsSlacker) [reply _posts/2019-11-22-one-year.md]: However - > whoever

Comment (MsSlacker) [reply _posts/2019-11-22-one-year.md]: his - > it's

File: _posts/2019-11-22-one-year.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+## Writing Statistics
+
+Now onto a few statistics for the year.
+
+[200WordsADay][11] says that I have written 175000 words, that's more than an epic fantasy novel right there. It clearly shows the compounding benefits of being consistent. Let's assume that you write only 200 words per day: that means 1.4k words a week or 6k words a month or 71k words a year. Not enough to qualify as an epic fantasy novel, but more than enough to be a novel in any other genre.
+
+I have written 338 articles since I started. That is a bit shy of the 356 articles for it to be an article a dat, but I have also written longer articles that took multiple days to write. I'm not including columns, pages, and email newsletters that don't really count as articles.
```
Comment (MsSlacker) [inline _posts/2019-11-22-one-year.md]: a dat - > a day

File: _posts/2019-11-22-one-year.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+Averaged that's 491 words a day which is more than double the 200 words a day threshold. I honestly thought that the number would be lower.
+<!-- It is astounding to see how several longer articles can push it up so much. -->
+
+## Products
+
+While writing I came up with an idea for a mobile app. I've always wanted somewhere to jot down ideas on the go, but I didn't like any single application that was available on [Android][15] for the task so I've written my own. The idea has changes slightly over time but the foundation was set. That's how Teria was born.
```
Comment (MsSlacker) [inline _posts/2019-11-22-one-year.md]: changes - > changed

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-22-one-year.md (hunk 3 above)
Comment (valeIT) [reply _posts/2019-11-22-one-year.md]: its maybe..?

**Pull request merged**